testdata/validate-data/validationResultTypes.csv

```diff
@@ -0,0 +1,10 @@
+Error Group,User Input Error Class,Main information from,Identifiable How?,sh:sourceConstraintComponent,sh:sourceShape,sh:detail - sh:sourceConstraintComponent,sh:detail - sh:sourceShape,sh:detail - sh:resultPath,res_id,res_type,prop_name,User message from,actual_type,actual_content

```

Comment:
We can delete it at the end. But while more shapes are added, this overview is helpful to keep track of everything.